tests/framework/extensions/provisioning/ssh.go

```diff
@@ -81,6 +95,35 @@ func CallSSHTestByName(testCase provisioninginput.SSHTestCase, node *nodes.Node,
 		}
 
 		return err
+
+	//This test checks if the audit log file is properly created on the node (skipped if its not a control plane node).
+	//For k3s you will need to configure the audit log dir to: /var/lib/rancher/k3s/etc/config-files/audit-policy-file
```

Comment:
is this not done via the code on line 106? If so I think the 2nd line of the comment is redundant.

Reply:
Agreed, it seems like 106 does this, so we can just remove this comment to avoid any confusion.